### PR TITLE
update changelog after v1.1.13 released

### PR DESCRIPTION
**Pull request by lifubang**

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
 
+## [1.1.13] - 2024-06-13
```
Comment (kolyshkin) [inline CHANGELOG.md:14]: I was thinking that versions should go in a sorted order, i.e. 1.2.0-rc should be above 1.1.13 (even if it's released earlier).

Comment (lifubang) [reply CHANGELOG.md:14]: I think both are ok, for my personal, I prefer to sort by timeline.

Comment (cyphar) [reply CHANGELOG.md:14]: I think version order makes more sense when reviewing when changes were added to each branch. If a user wants to see things in timeline order, they can look at https://github.com/opencontainers/runc/releases or just look at the dates.

**Pull request merged**